package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-advertising",
-  "version": "4.0.0",
+  "version": "4.0.1",
```

Comment:
Please leave the version at 4.0.0. Since we use `yarn`, it will be bumped when we publish the new version. (aside: since this is a new feature, it should have been version 4.1.0)